pandas/core/tools/datetimes.py

```diff
@@ -969,9 +965,6 @@ def to_datetime(
     ...                format='%Y-%m-%d %H:%M:%S.%f')
     Timestamp('2018-10-26 12:00:00.000000001')
 
-    :const:`"%S"` without :const:`"%f"` will capture all the way
-    up to nanoseconds if present as decimal places.
-
```

Comment:
you also need to remove the example code that follows

Reply:
Thank you for bringing this to my notice, I have deleted the example code.